filter.lua

```diff
@@ -112,29 +114,31 @@ local function filter(outbox)
 				outbox.notify(message)
 
 				if verdict.action == action.DROP then
-					outbox.reply(reply[dport] .. ":" .. tostring(packet))
+					outbox.reply(reply[dport] .. "|" .. tostring(packet))
```

Comment:
we could create a channel abstraction on top of mailboxes to encode/decode such message formats.. but I would leave this as an improvement for later and merge this PR as is..